features/mediation-features/data-publisher-features/org.wso2.micro.integrator.observability.feature/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>data-publisher-features</artifactId>
+        <groupId>org.wso2.ei</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.micro.integrator.observability.feature</artifactId>
+    <description>
+        This feature contains the core bundles required for publishing metrics data to Prometheus
```

Comment:
We can change the description to reflect that we have a generic observability feature.

Reply:
Fixed